mantidimaging/core/utility/version_check.py

```diff
@@ -94,15 +94,12 @@ def conda_update_message(self) -> tuple:
         msg += f".\nFound version {self.get_conda_installed_version()}, "
         msg += f"latest: {self.get_conda_available_version()}.\nPlease check the terminal for an update command!"
 
-        detailed = f"Not running the latest Mantid Imaging{suffix}.\n"
+        detailed = f"Not running the latest Mantid Imaging {suffix}.\n"
         detailed += f"Found version {self.get_conda_installed_version()}, "
         detailed += f"latest: {self.get_conda_available_version()}.\n"
         detailed += "To update your environment please copy and run the following command:\n"
-        detailed += "source /opt/miniconda/bin/activate /opt/miniconda && "
-        if self.get_conda_installed_label() != "main":
-            detailed += "ENVIRONMENT_NAME=mantidimaging_unstable REPO_LABEL=unstable "
-        detailed += "source "
-        detailed += "<(curl -s https://raw.githubusercontent.com/mantidproject/mantidimaging/master/install.sh)"
+        detailed += f"source /opt/miniconda/bin/activate /opt/miniconda/envs/mantidimaging_{suffix} && "
```

Comment:
This will fail for main on IDAAS because the environment is just `/opt/miniconda/envs/mantidimaging` . I think it would be best to just detect it from the environment in which MI is currently running.

`os.environ['CONDA_PREFIX']`

should always have the right value.

The same could be done for `/opt/miniconda/bin/activate`, looks like the best way to get it is using `CONDA_EXE` to get `/opt/miniconda/bin/conda` which is always in the same directory as the `activate` script.